README.md

```diff
@@ -34,6 +34,18 @@ for help in case of questions.
 
 Documentation for developers can be found at [DEVELOPING.md](DEVELOPING.md).
 
+## Development setup
+This app requires [Text](https://github.com/nextcloud/text) and [Circles](https://github.com/nextcloud/circles).
+You also need nvm v16.16.0 to compile the JavaScript. 
+After installing Text and Circles:
+
+1. Clone Collectives into the apps folder of your Nextcloud: 
+   `git clone https://github.com/nextcloud/circles`
```

Comment:
```suggestion
   `git clone https://github.com/nextcloud/collectives`
```

Reply:
jup embarrassing 🙈 thank you for the fast review :)